plugins/node/instrumentation-tedious/package.json

```diff
@@ -47,20 +47,23 @@
     "@opentelemetry/api": "^1.3.0"
   },
   "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "6.21.0",
+    "@typescript-eslint/parser": "6.21.0",
+    "eslint": "8.9.0",
     "@opentelemetry/api": "^1.3.0",
     "@opentelemetry/context-async-hooks": "^1.8.0",
     "@opentelemetry/contrib-test-utils": "^0.40.0",
     "@opentelemetry/sdk-trace-base": "^1.8.0",
     "@types/mocha": "7.0.2",
     "@types/node": "18.6.5",
-    "mocha": "7.2.0",
+    "mocha": "^10.4.0",
     "nyc": "15.1.0",
     "rimraf": "5.0.5",
     "semver": "7.6.0",
-    "tedious": "17.0.0",
+    "tedious": "18.2.0",
     "test-all-versions": "6.1.0",
-    "ts-mocha": "10.0.0",
-    "typescript": "4.4.4"
+    "ts-node": "^10.9.2",
+    "typescript": "5.3.3"
```

Comment:
I think this might be considered a break change which we try to avoid. @dyladan is it safe to apply?

Reply:
Why is that a breaking change? Its a dev dependency.

Please respect the reasoning in the PR description. We have to update ts

Reply:
I am not familiar with the exact details, but I remember discussions that if we update typescript version for the package, exiting users of the instrumentation with typescript 4 might have their transpiration broken.
@open-telemetry/javascript-maintainers can someone please help and supply guidance on the safe way to do that?